scripts/block-animations/block-animation.js

```diff
@@ -4,22 +4,37 @@
  * @description handles setting up animation settings for blocks
  *
  * theme.json settings:
- * "animation": [
- * 		{ "label": "Test", "value": "test" },
- * 		{ "label": "Test 2", "value": "test-2" }
+ * "animationType": [
```

Comment:
It might be nice to break this single file into smaller setting-specific partials and include/import them all together in an index/root file. This one is getting pretty long!

Reply:
@dpellenwood I updated this - mind taking another look? Obviously the main file is still going to be pretty large, but I think I like having the settings separate regardless. I also moved the attributes out. 

Maybe in the future it would be nice to pull this entire thing out of resources into it's own plugin - but that would be way down the line.